cli_github/mains.py

```diff
@@ -123,6 +126,18 @@ def main():
     if args.followers:
         name = url_parse(args.followers)
         url = GITHUB_API + 'users/' + name + '/followers'
+
+#FOLLOWING
+
+ 	if args.following :
```

Comment:
Lines 132-138 are throwing the error `inconsistent use of tabs and spaces in indentation`. Look into it.